runner.py

```diff
@@ -0,0 +1,32 @@
+from Instastories import start_scrape
+import argparse, random, time
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="TOKEN PATH AND NUMBER OF PEOPLE SCRAPED")
+    parser.add_argument("t", metavar="<token>", help="Insert the path of cookie file")
+    parser.add_argument("n", metavar="<number of persons>", type=int, help="Insert the number of people to scrape, use \"0\" to scrape all the users")
+    parser.add_argument("f", metavar="<folder path>", help="Insert the destination folder in which you want to download files")
+    parser.add_argument("m", metavar="<mode>", choices={"all", "media", "metadata"}, help="Set the scraping mode:\n all: scrape media and metadata\n media: scrape only media\n metadata: scrape only metadata")
+    parser.add_argument("l", metavar="<loop>", choices={"single", "loop"}, help="Use \"single\" for a single scraping cycle, \"loop\" for keep looping")
+    parser.add_argument("-d", metavar="<delay>", type=int, help="Set the delay between scraping sessions in \"loop\" mode")
+    parser.add_argument("-dp", metavar="<delay variance percentage>", type=int, help="Set minimum and maximum delay variance")
```

Comment:
It's not really a variance. But naming is hard, let's keep it this way. Set the default to 15.